.github/workflows/integration.yml

```diff
@@ -0,0 +1,74 @@
+name: integrations
```

Comment:
nit: shall we call it `integration` to match the file name?

Reply:
Ill name it to integration-test